onnxruntime/src/session.rs

```diff
@@ -111,6 +111,43 @@ impl<'a> SessionBuilder<'a> {
         Ok(self)
     }
 
+    /// Options settings to match the Ampere AIO examples. So far, the important one
+    /// is selecting the execution provider.
+    #[cfg(feature = "aio")]
+    pub fn with_aio_settings(self) -> Result<SessionBuilder<'a>> {
+        let mut ptr: *mut *mut ::std::os::raw::c_char = std::ptr::null_mut();
+        let mut len: i32 = 0;
+        unsafe {
+            g_ort().GetAvailableProviders.unwrap()(&mut ptr, &mut len);
+            g_ort().EnableCpuMemArena.unwrap()(self.session_options_ptr);
+            g_ort().EnableMemPattern.unwrap()(self.session_options_ptr);
+            g_ort().SetSessionExecutionMode.unwrap()(
+                self.session_options_ptr,
+                ExecutionMode::Sequential.into(),
+            );
+            g_ort().SetInterOpNumThreads.unwrap()(self.session_options_ptr, 1);
+        }
+
+        for i in 0..len {
+            unsafe {
+                let str = core::ffi::CStr::from_ptr(*ptr.offset(i as isize));
+                println!("{}: {}", i, str.to_string_lossy());
```

Comment:
are these printlns supposed to remain in here?

Reply:
I was thinking prints in prototypes code are ok, but we all know what happens to prototype code... Removed.